### PR TITLE
added appdynamics  to vendor

### PR DESCRIPTION
**Pull request by saurabhdes**
Added Appdynamics to vendor list.

### CODE REVIEW

**Review by julianocosta89 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ We welcome any vendor to fork the project to demonstrate their services and
 adding a link below. The community is committed to maintaining the project and
 keeping it up to date for you.
 
+- [Appdynamics](https://www.appdynamics.com/blog/cloud/how-to-observe-opentelemetry-demo-app-in-appdynamics-cloud/)
```
Comment (reyang) [inline README.md:55]: Should the `D` by uppercase?

image

Comment (reyang) [reply README.md:55]: image

Comment (saurabhdes) [reply README.md:55]: Thankyou @reyang for catching this ! Will create a fix soon.

Comment (saurabhdes) [reply README.md:55]: Created PR for the fix: https://github.com/open-telemetry/opentelemetry-demo/pull/595

**Pull request merged**